scripts/swupd-performance/samples/file_20190930_130123Z.json

```diff
@@ -0,0 +1,388 @@
+[
```

Comment:
is this file the result of an execution? why does it need to be under version control?

Reply:
Actually it is a sample output I created and cleaned to test out. We dont really need this once we an output that is 100% json. So i would like to keep a samples folder this then